Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1451,6 +1462,24 @@ def upload_file_command(client: Client, args: dict) -> tuple[str, dict, dict, bo
     return human_readable, entry_context, result, True
 
 
+def delete_attachment_command(client: Client, args: dict) -> tuple[str, dict, dict, bool]:
+    """Delete an attachment file.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+
+    Returns:
+        Demisto Outputs.
+    """
+    attachment_file_id = str(args.get('file_sys_id', ''))
+
+    result = client.delete_attachment(attachment_file_id)
+    if not result:
+        return f'Attachment with Sys ID {attachment_file_id} was successfully deleted.', {}, result, True
+    raise DemistoException("Error: No record found. Record doesn't exist or ACL restricts the record retrieval.")
```

Comment:
Are we sure we are covering all possible responses from the API?
Does `client.delete_attachment` actually returns empty `result` on a 204 response?
On error, "detail" and the "message" values under the “error” key from the response should be returned, I see a hard coded string instead of these values.

Reply:
Yes, in 204 we will receive null and all other bad requests are handled by the send_request function. The only possible error that returns is 404 (which it returns on purpose for mirroring according to what I understand)